src/main/java/com/elasticbox/Client.java

```diff
@@ -1153,8 +1175,28 @@ public boolean isTrusted(X509Certificate[] x509Certificates, String authType)
 
                 httpClientBuilder.setConnectionManager(connectionManager);
 
+                if (httpProxy != null) {
+                    if ( (!StringUtils.isBlank(httpProxy.host)) && (httpProxy.port != 0) ) {
+                        httpClientProxy = httpProxy;
+                        LOGGER.info("Proxy configured for connection through " + httpProxy.host + ":" + httpProxy.port);
+                        HttpHost proxyHostObject = new HttpHost(httpProxy.host, httpProxy.port);
+                        httpClientBuilder.setProxy(proxyHostObject);
+
+                        if (!StringUtils.isBlank(httpProxy.getUser()) && !StringUtils.isBlank(httpProxy.getPwrd())) {
+                            LOGGER.info("Proxy configured with credentials for " + httpProxy.getUser());
+                            CredentialsProvider credsProvider = new BasicCredentialsProvider();
+                            credsProvider.setCredentials(new AuthScope(httpProxy.host, httpProxy.port),
+                                    new UsernamePasswordCredentials(httpProxy.getUser(), httpProxy.getPwrd()));
+                            httpClientBuilder.setDefaultCredentialsProvider(credsProvider);
+                        }
+                    } else {
+                        LOGGER.warning("Misconfigured Jenkins proxy data. No proxy assumed." );
+                    }
+                }
+
                 httpClient = httpClientBuilder.build();
             } catch (Exception e) {
+                LOGGER.warning("Error while configuring httpClientBuilder." + e.getMessage() );
```

Comment:
Please log the complete Exception
```suggestion
                LOGGER.log(Level.WARNING, "Error while configuring httpClient.", e );
```